gdal2mbtiles/gdal.py

```diff
@@ -936,6 +936,7 @@ def render(self, outputfile, cmd=GDALTRANSLATE, working_memory=512,
                     '-q',                   # Quiet - FIXME: Use logging
                     '-of', 'GTiff',         # Output to GeoTIFF
                     '-co', 'BIGTIFF=IF_SAFER',  # Use BigTIFF if >2GB
+                    '-co', 'NUM_THREADS=ALL_CPUS', # multithreaded compression for GeoTiff
```

Comment:
Shouldn't the warp_cmd parameter here be something like "-mt" (for multithreading) instead of "-co"?

Reply:
As @mghughes helpfully pointed out... `-co` stands for "creation option" of which there can be multiple. Ignore my comment above.